examples/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
```suggestion
cmake_minimum_required (VERSION 3.13)
```

Reply:
Why the extra space?

Reply:
For consistency reasons - it has this space everywhere else too. But really not important, which is why have approved the PR anyway.